Processor/AbstractProcessor.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Pim\Bundle\EnhancedConnectorBundle\Processor;
+
+use Akeneo\Bundle\BatchBundle\Item\AbstractConfigurableStepElement;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * An abstract processor.
+ *
+ * @author    Damien Carcel <damien.carcel@akeneo.com>
+ * @copyright 2015 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class AbstractProcessor extends AbstractConfigurableStepElement
+{
+    /** @var NormalizerInterface */
+    protected $transNormalizer;
+
+    /**
+     * @param NormalizerInterface $transNormalizer
+     */
+    public function __construct(NormalizerInterface $transNormalizer)
+    {
+        $this->transNormalizer = $transNormalizer;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigurationFields()
+    {
+        return [];
+    }
+}
```

Comment:
Ihmo, it does not make sense to create this class only to inject TranslationNormalizer